client/layout/controller.js

```diff
@@ -60,6 +60,7 @@ export const getPages = () => {
 		],
 		wpOpenMenu: 'toplevel_page_woocommerce',
 		id: 'home',
+		navParent: 'woocommerce',
```

Comment:
I introduced the `id` parameter just for the purposes of the Nav so I'm confident its not used anywhere else. Given that, do you think it may be worthwhile to just make the id's themselves more unique, ie `id: 'woocommerce/home'`?

Reply:
+1

Reply:
Definitely :+1: